lib/cylc/cylc_xdot.py

```diff
@@ -401,9 +402,18 @@ def get_graph(self, group_nodes=[], ungroup_nodes=[],
         graph.graph_attr['rankdir'] = self.orientation
 
         for node in graph.nodes():
-            name = TaskID.split(node.get_name())[0]
+            name, point = TaskID.split(node.get_name())
             if name in family_nodes:
+                # Style family nodes.
                 node.attr['shape'] = 'doubleoctagon'
+            else:
+                # Style regular (task) nodes.
+                node.attr['shape'] = 'oval'
+            if not any(seq.is_on_sequence(get_point(point)) for seq in
+                       self.suiterc.taskdefs[name].sequences):
```

Comment:
In essence of our recent validate optimisation, we know that sequence calculations are expensive. I wonder if it is worth caching the results here?